change/@fluentui-react-storybook-addon-export-to-sandbox-e826cc4e-1629-4a56-b91e-864b8456f4c1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
this is actually a breaking change as we bump minimun SB support, please update `type`